webapp-django/crashstats/manage/admin.py

```diff
@@ -100,6 +104,18 @@ def supersearch_fields_missing(request):
     return render(request, "admin/supersearch_fields_missing.html", context)
 
 
+@superuser_required
+def supersearch_status(request):
+    context = {}
+    status = SuperSearchStatus().get()
+    context["indices"] = status["indices"]
+    context["latest_index"] = status["latest_index"]
+    context["mapping"] = json.dumps(status["mapping"], indent=2)
```

Comment:
This is a cheap/easy way to make the structure readable in the web page. Django has a `json_script` template filter, but that puts it in `<script>` tags and we don't want to do that.